src/Orchard.Recipes.Abstractions/Services/IRecipeExecutor.cs

```diff
@@ -1,10 +1,10 @@
-﻿using Orchard.Recipes.Models;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
+using Orchard.Recipes.Models;
 
 namespace Orchard.Recipes.Services
 {
     public interface IRecipeExecutor
     {
-        Task<string> ExecuteAsync(string executionId, RecipeDescriptor recipeDescriptor);
+        Task<string> ExecuteAsync(string executionId, RecipeDescriptor recipeDescriptor, object environment);
```

Comment:
not sure how I feel about passing just a random object through.... it feel like it allows the user to place fast and loose with the execute method.